src/main/scala/org/deepdive/settings/SettingsParser.scala

```diff
@@ -1,253 +1,536 @@
 package org.deepdive.settings
```

Comment:
Is it better to combine the settings in one file or in separate files? It seems after combining, this file is getting big and comprehensive.